src/main/java/it/pagopa/selfcare/pagopa/backoffice/controller/BrokerController.java

```diff
@@ -115,12 +146,36 @@ public ResponseEntity<Resource> exportCreditorInstitutionToCsv(@Parameter(descri
                 .body(new ByteArrayResource(file));
     }
 
-    @GetMapping(value = "/{broker-code}/export-status", produces = {MediaType.APPLICATION_JSON_VALUE})
+    @GetMapping(value = "/{broker-code}/export-status",
+            produces = {MediaType.APPLICATION_JSON_VALUE})
     @ResponseStatus(HttpStatus.OK)
-    @Operation(summary = "Get all info about data exports for the broker EC", security = {@SecurityRequirement(name = "JWT")})
+    @Operation(summary = "Get all info about data exports for the broker EC",
+            security = {@SecurityRequirement(name = "JWT")})
     @OpenApiTableMetadata(readWriteIntense = OpenApiTableMetadata.ReadWrite.READ)
-    public BrokerECExportStatus getBrokerExportStatus(@Parameter(description = "SelfCare Broker Code. it's a tax code") @PathVariable("broker-code") String brokerCode) {
-
+    public BrokerECExportStatus getBrokerExportStatus(
+            @Parameter(description = "SelfCare Broker Code. it's a tax code") @PathVariable("broker-code") String brokerCode
+    ) {
         return exportService.getBrokerExportStatus(brokerCode);
     }
+
+    /**
+     * Retrieves the list of creditor institution's delegation for the specified broker
+     * 
+     * @param brokerId the broker identifier
+     * @param brokerCode the broker tax code
+     * @return the list of broker's delegations
+     */
+    @GetMapping("/{broker-code}/delegations")
+    @ResponseStatus(HttpStatus.OK)
+    @Operation(
+            summary = "Retrieve all delegations for given ci broker",
+            security = {@SecurityRequirement(name = "JWT")})
+    @OpenApiTableMetadata(readWriteIntense = OpenApiTableMetadata.ReadWrite.READ, cacheable = true)
+    @Cacheable(value = "getBrokerDelegation")
+    public List<MyCIResource> getBrokerDelegation(
+            @Parameter(description = "Broker's tax code") @PathVariable("broker-code") String brokerCode,
+            @Parameter(description = "Broker's unique id") @RequestParam String brokerId
```

Comment:
I dont' understand. The broker identifier is its tax code, so what's the difference between `broker-code` and `broker-id`?

Reply:
the `broker-code` is the broker tax code while the `broker-id` is `the selectedParties.id`` in the redux context, i suppose is the selfcare id.